docs/HOOKS.md

```diff
@@ -102,6 +106,27 @@ Following is the list of hooks supported by Web Chat API.
 -  [`useVoiceSelector`](#usevoiceselector)
 -  [`useWebSpeechPonyfill`](#usewebspeechponyfill)
 
+## `useActiveTyping`
+
+```js
+interface Typing {
+  at: number;
+  expireAt: number;
+  name: string;
+  role: 'bot' | 'user';
+}
+
+useActiveTyping(expireAfter?: number): [{ [id: string]: Typing }]
+```
+
+This function will return a list of participants who are actively typing, including the start typing time (`at`) and expiration time (`expireAt`), the name and the role of the participant.
+
+If the participant send a message after the typing activity, the participant will be explicitly removed from the list. If no message or typing activity is received, the participant is considered inactive and not listed in the result. To keep typing indicator active, participants should send typing activity continuously.
+
+The inactivity timer can be overridden by passing the `expireAfter` argument. If `Infinity` is passed, it will return all participants who did not explicitly remove from the list. In other words, it will return participants who sent typing activity, but did not send a message activity afterward.
```

Comment:
```suggestion
The `expireAfter` argument can override the inactivity timer. If `expireAfter` is `Infinity`, it will return all participants who did not explicitly remove from the list. In other words, it will return participants who sent a typing activity, but did not send a message activity afterward.
```

Reply:
Love the reversion of words makes the sentence much cleaner.